src/containers/projects/Projects.js

```diff
@@ -1,13 +1,16 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, lazy, Suspense } from "react";
 import ApolloClient from "apollo-boost";
 import { gql } from "apollo-boost";
 import "./Project.css";
-import GithubRepoCard from "../../components/githubRepoCard/GithubRepoCard";
 import Button from "../../components/button/Button";
+import Loading from "../loading/Loading";
 import { openSource, socialMediaLinks } from "../../portfolio";
-import { Fade } from "react-reveal";
+
 
 export default function Projects() {
+  const GithubRepoCard = lazy(() => import('../../components/githubRepoCard/GithubRepoCard'));
+  const FailedLoading = () => <div className="centerContent"><h3>Looks like you have not configured for Github Repo.</h3></div>;
```

Comment:
We don't have to force the user to show his gh projects, lets return null if not found, and to help debug, we can log "Projects section not configured" or something.